templates/bundles/DemosPlanCoreBundle/DemosPlanNews/globalnewslist_list_index.html.twig

```diff
@@ -6,8 +6,10 @@
     }%}
 
         {% block content %}
-
-            <div class="layout__item u-1-of-1-palm{% if hidePressAnnouncements is not defined %} u-1-of-2-lap-up{% endif %} u-pl-0_5 u-pl-0-lap-up u-pr-lap-up">
+            <div class="layout__item u-1-of-1-palm  u-pl-0_5 u-pl-0-lap-up u-pr-lap-up
+                {% if templateVars.pressList is not empty %}
```

Comment:
`hidePressAnnouncements` is still used within `globalnewslist_list_index.html.twig`. To keep the current behavior we could chain the two conditions like so:

```suggestion
                {% if templateVars.pressList is not empty or hidePressAnnouncements is not defined %}
```

Reply:
its not needed this way. look at PR https://github.com/demos-europe/demosplan-project-blp/pull/38